eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageTypes.scala

```diff
@@ -560,6 +560,7 @@ case class GossipTimestampFilter(chainHash: ByteVector32, firstTimestamp: Timest
 
 case class OnionMessage(blindingKey: PublicKey, onionRoutingPacket: OnionRoutingPacket, tlvStream: TlvStream[OnionMessageTlv] = TlvStream.empty) extends LightningMessage
 
+case class Stfu(channelId: ByteVector32, initiator: Byte) extends LightningMessage
```

Comment:
nit: should be right before `SpliceInit`

Reply:
Fixed in [9fa905a](https://github.com/ACINQ/eclair/pull/2680/commits/9fa905a3e557420b011b8c5079cb48ba74b705a2)